src/react-components/ui-root.js

```diff
@@ -1391,7 +1397,8 @@ class UIRoot extends Component {
                         )}
                         <PeopleMenuButton
                           active={this.state.sidebarId === "people"}
-                          onClick={() => this.toggleSidebar("people")}
+                          disabled={isLockedDownDemo}
```

Comment:
This looks appropriate to me. @nickgrato ?

Reply:
@nikk15 are you asking about the disabled prop?

Reply:
yea looks like it drills down to the <button> eventually and disabled is an accurate button attr  🤷‍♂️